src/ontology/uberon.Makefile

```diff
@@ -49,8 +55,7 @@ update_dynamic_catalog:
 	@echo "WARNING: files are expected to be available. Do not do this if you dont"
 	@echo "WARNING: know what you are doing."
 	$(SCRIPTSDIR)/make-catalog.pl uberon.owl mirror/ncbitaxon.owl imports/*_import.owl \
-		mirror/ro.owl imports/local-*owl $(BRIDGEDIR)/*owl $(TMPDIR)/allen-*.obo \
-		$(TMPDIR)/developmental-stage-ontologies/src/ssso-merged.obo > $@.tmp && \
```

Comment:
What is the story behind removing `ssso-merged.obo`?

Reply:
I somehow lost touch of this part of the development - so we don't need the ssso in the process at all anymore?

Reply:
We still include the SSSO in `composite-metazoan`, but instead of doing the weird things that we used to do (cloning the entire repository in `$(TMPDIR)` and using it from there), we treat it the same way we treat all the other foreign ontologies:

* mirror it into `$(MIRRORDIR)`;
* create a “local import” module in `$(IMPORTDIR)/local-ssso.owl`, which is committed to the repository;
* use `$(IMPORTDIR)/local-ssso.owl` whenever we need it that ontology (such as when building `composite-metazoan`).

The only special thing about SSSO compared to the other foreign ontologies is that for the mirroring step, we get the ontology directly from its GitHub source instead of going through `purl.obolibrary.org`, because the SSSO does _not_ have a PURL configuration. (By the way it should have one IMO, but that’s another issue.)